usrtools/txgen/app/cli-functions.c

```diff
@@ -504,6 +588,40 @@ misc_cmd(int argc, char **argv)
     return 0;
 }
 
+static struct cli_map page_map[] = {{10, "page %d"},
+                                    {11, "page "
+                                         "%|main|pcap|latency"},
```

Comment:
This line should be on line 592. Could use 'lat' as a shorthand name.

Reply:
I will fix it